### PR TITLE
Customizable signup modal title

### PR DESCRIPTION
**Pull request by ali-hosseini-fs**

### CODE REVIEW

File: packages/core/src/Stores/ui-store.js [hunk 1]
```diff
@@ -411,10 +412,11 @@ export default class UIStore extends BaseStore {
     }
 
     @action.bound
-    openRealAccountSignup(target = this.root_store.client.upgradeable_landing_companies?.[0]) {
+    openRealAccountSignup(target = this.root_store.client.upgradeable_landing_companies?.[0], callSite) {
```
Comment (reza-fs) [inline packages/core/src/Stores/ui-store.js:415]:
```suggestion
    openRealAccountSignup(target = this.root_store.client.upgradeable_landing_companies?.[0], call_site) {
```

File: packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx [hunk 2]
```diff
@@ -69,7 +75,7 @@ const WizardHeading = ({ real_account_signup_target, currency, is_isle_of_man_re
                 country_standpoint.is_other_eu ||
                 country_standpoint.is_rest_of_eu
             ) {
-                return <Localize i18n_default_text='Add a real Deriv Multipliers account' />;
+                return <Localize i18n_default_text={`Add a real Deriv ${signup_modal_real_account_title} account`} />;
```
Comment (reza-fs) [inline packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx:78]:
```suggestion
                return <Localize i18n_default_text={`Add a ${signup_modal_real_account_title} account`} />;
```

File: packages/core/src/Stores/ui-store.js [hunk 3]
```diff
@@ -411,10 +412,11 @@ export default class UIStore extends BaseStore {
...
         this.is_real_acc_signup_on = true;
         this.real_account_signup_target = target;
         this.is_accounts_switcher_on = false;
+        if (callSite) this.signup_modal_real_account_title = callSite;
```
Comment (reza-fs) [inline packages/core/src/Stores/ui-store.js:419]:
```suggestion
        this.signup_modal_real_account_title = call_site || this.signup_modal_ral_account_title;
```

Comment (ali-hosseini-fs) [reply packages/core/src/Stores/ui-store.js]: Done

File: packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx (hunk 2 above)
Comment (ali-hosseini-fs) [reply packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx]: Done

**Pull request merged**